downstream/modules/platform/proc-controller-configure-usability-analytics.adoc

```diff
@@ -0,0 +1,25 @@
+[id="controller-configure-usability-analytics"]
+
+= Configuring usability analytics and data collection
+
+Usability data collection is included with {ControllerName} to collect data to better understand how users specifically interact with it, to help enhance future releases, and to continue streamlining your user experience.
```

Comment:
to collect data to understand how users interact with it, to enhance future releases, and to streamline your user experience.